examples/darwin-framework-tool/commands/delay/WaitForCommissioneeCommand.mm

```diff
@@ -66,11 +33,16 @@ - (void)device:(MTRDevice *)device receivedEventReport:(NSArray<NSDictionary<NSS
     }
     base_device = nil;
 
-    auto * device = [MTRDevice deviceWithNodeID:@(mNodeId) controller:commissioner];
-    VerifyOrReturnError(device != nil, CHIP_ERROR_INCORRECT_STATE);
-
     auto queue = dispatch_queue_create("com.chip.wait_for_commissionee", DISPATCH_QUEUE_SERIAL);
-    [device setDelegate:mDeviceDelegate queue:queue];
+    [commissioner getBaseDevice:mNodeId
```

Comment:
I guess we can do this, but adding more use of deprecated APIs that we are trying to remove is not great....  If we just want to do a CASE setup attempt, why wouldn't we just do a single read on an MTRBaseDevice that we get the non-deprecated way, which will trigger CASE, etc?

Reply:
I have updated the code to basically do the same thing than what the current `TestCommandBridge.h` code is doing. It should not use `getBaseDevice`.